code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/TokenUtils.java

```diff
@@ -104,6 +95,25 @@ public Account getAccountFromJWT() {
         return jsonData;
     }
 
+    private String removeBearer(String jwtKey) {
+        String toParse;
+        if (StringUtils.isBlank(jwtKey)){
+            return jwtKey;
+        }
+        String[] tokenArr = jwtKey.split("\\s+");
+        if (tokenArr.length == 2) {
+            if (!StringUtils.equalsIgnoreCase("bearer", StringUtils.trim(tokenArr[0]))) {
+                return null;
```

Comment:
If we return null, we're passing that null to retrieveJwt(toParse) which ends up passing it to authTokenDao.getTokenByKey(jwtKey) which ends up passing a null into an SQL query